### DISCUSSION TITLE
trapped out from admin page due to wrong email configuration

### DISCUSSION THREAD

**Discussion by johanesmikhael** [mozilla/hubs#5703] [Hubs Cloud Questions] [answered]
Hello all,

Sorry for the stupid question but currently I cannot found the solution:

I was trying to set up an SMTP via send grid for an instance I deployed in AWS server, but eventually, it doesn't work and I lost my admin credential and cannot change it back to the default setting (since my instance couldn't send the magic link to me due to wrong setting). How to fix this problem? Thank you in advance

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-551)

**Comment by brianpeiris** [comment, marked as answer]
This question was asked a while ago, so it may no longer be useful to the original reporter, but I imagine we would resolve this kind of issue by updating the AWS CloudFormation stack with the correct admin email, or manually updating the database (via the elixir console) with the correct email address.

**Reply by pattersonbl2** [reply]
This would be the correct way because I configured my stack with the wrong email and could easily change it to work email by updating the template.
